src/fixtures.ts

```diff
@@ -1281,23 +1281,28 @@ const MULTISIGS_BASE = [
 ];
 
 const MULTISIGS = MULTISIGS_BASE.map((test) => {
+  let braidAwareMultisig = {};
   const multisig = generateMultisigFromPublicKeys(
     test.network,
     test.type,
     2,
     ...test.publicKeys
   );
-  multisig.braidDetails = braidConfig(test.braidDetails);
-  multisig.bip32Derivation = test.bip32Derivation;
+  braidAwareMultisig = {
```

Comment:
is this defined as a type anywhere? (sorry still reviewing but could be useful)

Reply:
No. The "Multisig" type was only previously typed by jsdoc and even then it never had a defined shape. I think further work could be done to make it a real type, but it didn't fit in the scope of this upgrade. It seems like it would take some discovery.